src/docs/product/integrations/source-code-mgmt/github/index.mdx

```diff
@@ -231,6 +231,22 @@ This feature will be automatically enabled for you once your GitHub integration
 
 ![How to disable Sentry comments on GitHub pull requests](github-org-settings.png)
 
+### Missing Member Detection
+
+If your organization has users committing to Github repositories linked to Sentry that are not members of your organization, Sentry will remind organization owners and managers to invite these users through a banner on the members list page.
+
+![Github missing members invite banner](github-missing-members-banner.png)
+
+The banner will show the 5 most active contributors. To see all of the missing members (up to 50), click **View All** on the banner. To snooze the banner, click the ellipsis > **Hide Missing Members**.
+
+// TODO: modal image
+
+You can select multiple users and assign roles and teams before inviting them through the modal.
+
```

Comment:
there is also a nudge email that will be sent out once a month to org owners and managers that haven't disabled this feature. should i include it here?

Reply:
Suggested putting this in the first sentence